src/ape/api/providers.py

```diff
@@ -860,6 +864,9 @@ class SubprocessProvider(ProviderAPI):
     process: Optional[Popen] = None
     is_stopping: bool = False
 
+    stdout_queue: Optional[JoinableQueue] = None
```

Comment:
At first tried naming these `_stdout_queue` etc.. but then Hardhat complains about the base classes conflictling.
Lol!